new-dti-website/components/apply/ApplicationTimeline.tsx

```diff
@@ -14,13 +14,16 @@ type TabProps = {
   isSelected: boolean;
   text: string;
   onClick?: () => void;
+  isSingleTab?: boolean;
 };
 
-const Tab: React.FC<TabProps> = ({ isSelected, text, onClick }) => (
+const Tab: React.FC<TabProps> = ({ isSelected, text, onClick, isSingleTab }) => (
   <button
     className={`flex items-center lg:px-5 lg:py-4 md:px-4 md:py-3 xs:px-2 md:rounded-t-xl xs:rounded-t-lg ${
       isSelected ? 'bg-[#FEFEFE] text-[#A52424]' : 'text-[#FEFEFE]'
-    } hover:cursor-pointer md:h-min xs:h-full`}
+    } hover:cursor-pointer md:h-min xs:h-full ${
```

Comment:
Question: Do you think we even need any text here at all if it's just one timeline? I'm curious how we'd justify putting that text since clearly the timeline is for all students.

Reply:
Honestly i dont think it hurts to add the "All students" label. It makes it clear and explicit that everyone has the same dates.

Plus, for those who might've applied in the Fall and then got rejected, if they come back in Spring and see that there aren't any tabs for Freshmen/Upperclassmen, it might be confusing to them